newrelic/python_agent.go

```diff
@@ -0,0 +1,71 @@
+package newrelic
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/buildpacks/libcnb"
+	"github.com/paketo-buildpacks/libpak"
+	"github.com/paketo-buildpacks/libpak/bard"
+	"github.com/paketo-buildpacks/libpak/effect"
+	"github.com/paketo-buildpacks/libpak/sherpa"
+)
+
+type PythonAgent struct {
+	BuildpackPath    string
+	ApplicationPath  string
+	Executor         effect.Executor
+	LayerContributor libpak.DependencyLayerContributor
+	Logger           bard.Logger
+}
+
+func NewPythonAgent(applicationPath string, buildpackPath string, dependency libpak.BuildpackDependency, cache libpak.DependencyCache) (PythonAgent, libcnb.BOMEntry) {
```

Comment:
```suggestion
func NewPythonAgent(applicationPath string, buildpackPath string) (PythonAgent, libcnb.BOMEntry) {
```

Sticking with previous comments, these are unnecessary.

Reply:
Done